.github/CONTRIBUTING.md

```diff
@@ -71,9 +71,12 @@ Pull Requests complies with these rules:
 ## Testing
 
 Before submitting a PR please verify that your code passes all the tests. To do
-so run the `unit-tests()` function along with any other function specified by the
-documentation in the [tests file][Tests file].
-
+that copy-paste the content of `unit-tests.gs` to append it to the `code.gs` in the
+google script-editor, then run `unitTests()`. It is good to also run `test()` with
```

Comment:
I'd mention that you can both append `unit-tests.gs` to `code.gs` or create a new script file in the project and place the contents of `unit-tests.gs` there.